upgrades/2024_02_29_131833_populate_update_settings_menu.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+use Illuminate\Support\Facades\DB;
+use ProcessMaker\Models\Setting;
+use ProcessMaker\Models\SettingsMenus;
+use ProcessMaker\Upgrades\UpgradeMigration as Upgrade;
+
+class PopulateUpdateSettingsMenu extends Upgrade
+{
+    /**
+     * Run any validations/pre-run checks to ensure the environment, settings,
+     * packages installed, etc. are right correct to run this upgrade.
+     *
+     * Throw a \RuntimeException if the conditions are *NOT* correct for this
+     * upgrade migration to run. If this is not a required upgrade, then it
+     * will be skipped. Otherwise the exception thrown will be caught, noted,
+     * and will prevent the remaining migrations from continuing to run.
+     *
+     * Returning void or null denotes the checks were successful.
+     *
+     * @return void
+     *
+     * @throws \RuntimeException
+     */
+    public function preflightChecks()
+    {
+        // Rename some values in the column 'group'
+        Setting::where('group', 'External Integrations')->update(['group' => 'Enterprise Integrations']);
```

Comment:
In some places the group are used as a key: https://github.com/search?q=org%3AProcessMaker%20%22External%20Integrations%22&type=code

Consider to change the translation to avoid side effects.

Reply:
I created a specific ticket to confirm this change, because the label "Users" is used in a lot of places
`"Users": "Additional Information" `is not able.